api/run_import.py

```diff
@@ -1,17 +1,18 @@
 import logging
 import os
+
 import django
 
+from datasets.blackspots.models import Document, Spot  # noqa
+from import_process.clean import clear_models  # noqa
+from import_process.process_xls import process_xls  # noqa
 from objectstore_interaction.connection import get_blackspots_connection
 from objectstore_interaction.fetch_spots import fetch_spots
 from objectstore_interaction.list_documents import get_documents_list
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "settings.settings")
 django.setup()
 
-from datasets.blackspots.models import Spot, Document  # noqa
```

Comment:
ew

Reply:
Currently not in the scope of this PR to change the actual imports.